### PR TITLE
Development

### PR DESCRIPTION
**Pull request by deusalexmachina**
Merging dev branch into master before code diverges too much. Can someone review? this appears to work with meghana's task routes. Thanks!

### CODE REVIEW

File: task_manager/api.py [hunk 1]
```diff
@@ -14,20 +14,25 @@
 from .routes.users import model_fetch_user, model_get_update_user
 from .routes.users import model_post_update_user, model_delete_user
 
+from flask_restx import Resource, Namespace
```
Comment (SemicolonExpected) [inline task_manager/api.py:17]: F811: redefinition of unused 'Resource' from line 6

**Review by SemicolonExpected — CHANGES_REQUESTED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Comment by SemicolonExpected** [comment]
Why did you change the keys from an env variable to a string?

**Comment by deusalexmachina** [comment]
> Why did you change the keys from an env variable to a string?

We should have a consistent secret key for when we test login and sessions, otherwise we will have issues when someone tries to decrypt something that was encoded using someone else's local variable. We can generate a new server-side key when we are finished.

**Comment by SemicolonExpected** [comment]
>  We can generate a new server-side key when we are finished.

I'm a bit confused since even after we're finished in theory it would still be worked on. Wouldn't it make more sense to have 2, one for testing (that you use a github secret for) and one that you set in a .env that is passed around for people to play with?

**Comment by deusalexmachina** [comment]
> I'm a bit confused since even after we're finished in theory it would still be worked on. Wouldn't it make more sense to have 2, one for testing (that you use a github secret for) and one that you set in a .env that is passed around for people to play with?

We do in essence have two options. the secret_key is for dev production. and uncommenting a line allows us to load from .env

**Comment by SemicolonExpected** [comment]
I guess my question is more of should we not also save the plantext key in the env file and just switch it out (via comment/uncomment) when we switch from dev to prod built? Just in case we have to use it elsewhere

**Comment by deusalexmachina** [comment]
> I guess my question is more of should we not also save the plantext key in the env file and just switch it out (via comment/uncomment) when we switch from dev to prod built? Just in case we have to use it elsewhere

I added a new commit, default loads from .env, running in dev replaces with the test key.

**Pull request merged**